src/main/scala/org/broadinstitute/dsde/firecloud/service/ShareLogService.scala

```diff
@@ -9,6 +9,8 @@ import org.broadinstitute.dsde.firecloud.model.UserInfo
 import org.broadinstitute.dsde.firecloud.service.PerRequest.{PerRequestMessage, RequestComplete}
 import org.broadinstitute.dsde.firecloud.service.ShareLogService.{Autocomplete, GetShares, LogShare}
 import org.broadinstitute.dsde.rawls.model.{AttributeFormat, PlainArrayAttributeListSerializer}
+import spray.httpx.SprayJsonSupport
+import spray.json.DefaultJsonProtocol._
```

Comment:
`SprayJsonSupport` handles un/marshalling from http classes; `DefaultJsonProtocol` handles de/serialization of the basics (Strings, Collections, etc)